host/sgx/create.c

```diff
@@ -874,6 +874,27 @@ oe_result_t oe_sgx_build_enclave(
         &loaded_enclave_pages_size,
         &enclave_size));
 
+    if (context->is_config_id_setting_mandatory && !(props.config.attributes & OE_SGX_FLAGS_KSS))
+    {
+        // Fail if the OE_SGX_FLAGS_KSS is not set when config_id/config_svn
+        // settings are mandatory
+        OE_RAISE_MSG(
+            OE_UNSUPPORTED,
+            "Enclave image requires config_id/config_svn settings but "
+            "OE_SGX_FLAGS_KSS is not set in properties\n",
+            NULL);
+    }
+
+    if(context->is_config_id_setting_mandatory && !_is_kss_supported())
```

Comment:
Nit: Missing space between `if(` 
If you haven't already I suggest running it on the changelist to prevent CI from blocking it when you want to merge the changes.